### ISSUE TITLE
Land upstream copy/paste patch in our fork

### ISSUE THREAD

**Issue by sedge** [humphd/brackets#44]
Brackets isn't enjoying when we completely overwrite the default html with new source code. It breaks hard and stops the preview process.

The console errors look like this:

```js
data-brackets-id=8 not found
data-brackets-id=13 not found
data-brackets-id=12 not found
```

**Comment by sedge** [comment]
Those errors are on the blob html file too, so it looks like it needs to be regenerated somehow.

**Comment by sedge** [comment]
Our testing is showing that brackets has an issue with this in the standard live development functionality as well as the experimental multi-browser version.

**Comment by gideonthomas** [comment]
This doesn't seem to be an issue anymore.

Except, it now manifests as https://github.com/adobe/brackets/issues/10634

**Comment by humphd** [comment]
...which has a patch written already https://github.com/adobe/brackets/pull/10653 :)  We'll rebase when that lands, and all should be well.

**Comment by sedge** [comment]
@humphd Are there any other considerations we need to make if we land this on our fork?

**Comment by gideonthomas** [comment]
Fixed with e81907d6c9b8c609127d5c9aadc809dd4efa1984
